### ISSUE TITLE
Export, then Import Fails

### ISSUE THREAD

**Issue by briangreunke** [CTFd/CTFd#1628]
**Environment**:

- CTFd Version/Commit: 3.0.2/c1d7910
- Operating System: Docker version 19.03.12, build 48a66213fe, docker-compose version 1.26.2, build eefe0d31, Mac OS 10.15.5, (Also tested Ubuntu 18.04 and w/o Docker)
- Web Browser and Version: Firefox 79.0/Chrome Version 84.0.4147.135

**What happened?**
Import fails (hangs) AFTER executing an export. A request to the `/admin/import` route never occurs.

The progress bar appears on screen, no request is sent to the application, and the import hangs.

**What did you expect to happen?**
Import should succeed. Successful debug logs appear.

```
ctfd_1   | INFO  [alembic.runtime.migration] Context impl MySQLImpl.
ctfd_1   | INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
ctfd_1   | /usr/local/lib/python3.7/site-packages/pymysql/cursors.py:170: Warning: (1280, "Name 'alembic_version_pkc' ignored for PRIMARY key.")
ctfd_1   |   result = self._query(query)
ctfd_1   | INFO  [alembic.runtime.migration] Running upgrade  -> 8369118943a1, Initial Revision
ctfd_1   | INFO  [alembic.runtime.migration] Running upgrade 8369118943a1 -> 4e4d5a9ea000, Add type to awards
ctfd_1   | INFO  [alembic.runtime.migration] Running upgrade 4e4d5a9ea000 -> b5551cd26764, Add captain column to Teams
ctfd_1   | INFO  [alembic.runtime.migration] Running upgrade b5551cd26764 -> b295b033364d, Add ondelete cascade to foreign keys
ctfd_1   | INFO  [alembic.runtime.migration] Running upgrade b295b033364d -> 080d29b15cd3, Add Tokens table to store user access tokens
ctfd_1   | INFO  [alembic.runtime.migration] Running upgrade 080d29b15cd3 -> a03403986a32, add theme code injections to configs
ctfd_1   | INFO  [alembic.runtime.migration] Running upgrade a03403986a32 -> 1093835a1051, Add default email templates
ctfd_1   | INFO  [alembic.runtime.migration] Context impl MySQLImpl.
ctfd_1   | INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
ctfd_1   | INFO  [alembic.runtime.migration] Context impl MySQLImpl.
ctfd_1   | INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
ctfd_1   | INFO  [alembic.runtime.migration] Context impl MySQLImpl.
ctfd_1   | INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
ctfd_1   | 172.21.0.1 - - [01/Sep/2020:16:24:55 +0000] "POST /admin/import HTTP/1.1" 302 233 "http://localhost:8000/admin/config" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:79.0) Gecko/20100101 Firefox/79.0"
ctfd_1   | 172.21.0.1 - - [01/Sep/2020:16:24:55 +0000] "GET /admin/config HTTP/1.1" 302 271 "http://localhost:8000/admin/config" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:79.0) Gecko/20100101 Firefox/79.0"
```

**How to reproduce your issue**
Issue occurs in multiple configurations, but easily reproducible in the `docker-compose` configuration.
- `docker-compose up`
- Complete setup
- Complete export
- Attempt to import the previously exported `db.zip`

**Any associated stack traces or error logs**
- Request to export:
```
ctfd_1   | 172.21.0.1 - - [01/Sep/2020:16:18:28 +0000] "GET /admin/export HTTP/1.1" 200 0 "http://localhost:8000/admin/config" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:79.0) Gecko/20100101 Firefox/79.0"
```

- Request to import:
(no logs generated)

**Notes**: 
- the import process succeeds if it is conducted BEFORE an export, or if the application is restarted and import is executed again.
- also verified in a deployment w/ external MySQL/Redis instances

**Comment by ColdHeat** [comment]
I've theorized that maybe you can subprocess a new Python interpreter and run the import from there and perhaps avoid some deadlocks that could be occurring here. Maybe that's something you can prototype to see if it fixes the issue.

**Comment by ColdHeat** [comment]
This should be fixed in the 3.2.0-dev branch.
